ros2node/ros2node/verb/get_type_description.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2023 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import rclpy
+from ros2cli.node import NODE_NAME_PREFIX
+from ros2cli.node.strategy import add_arguments
+from ros2node.api import (
+    NodeNameCompleter,
+    parse_node_name,
+)
+from ros2node.verb import VerbExtension
+from type_description_interfaces.msg import FieldType
+from type_description_interfaces.srv import GetTypeDescription
+
+
+def print_field_type(ft):
+    type_name = None
+    type_str = ''
+    for x, y in FieldType.__dict__.items():
+        if y == ft.type_id:
+            type_name = x
+            break
+    if not type_name:
+        raise RuntimeError(f'Could not match FieldType type_id {ft.type_id} to a name')
+
+    type_str = type_name
+    if 'NESTED_TYPE' in type_name:
+        type_str += f' ({ft.nested_type_name})'
+    if ft.string_capacity:
+        type_str += f' - string_capacity {ft.string_capacity}'
+    if ft.capacity:
+        type_str += f' - capacity {ft.capacity}'
```

Comment:
```suggestion
    if ft.string_capacity:
        type_str += f' string_capacity({ft.string_capacity})'
    if ft.capacity:
        type_str += f' capacity({ft.capacity})'
```
IMO formatting with `- capacity 255` could be a bit confusing especially if afterward will be printed out the dafult value. It is gonna look like `FIELD_TYPE_INT8_ARRAY - capacity 25 = 0`